spec/controllers/api/v1/employees_controller_manager_spec.rb

```diff
@@ -1,12 +1,10 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
```

Comment:
why this line was removed? all rspec configuration stuff should be in rails_helper...